src/resources/views/create.blade.php

```diff
@@ -181,6 +181,10 @@ class="btn btn-success"
             They already exist in another storage request with equal directory name.
         </p>
 
+        <p v-cloak v-if="hasTIFFfile" class="text-info">
```

Comment:
```suggestion
        <p v-cloak v-if="hasTIFFfile" class="text-warning">
```
Should be a `text-warning` (see #22).